client/web/web.go

```diff
@@ -97,6 +98,21 @@ func NewServer(ctx context.Context, opts ServerOpts) (s *Server, cleanup func())
 	return s, cleanup
 }
 
+// debugMode returns the debug mode the web client is being run in.
+// The empty string is returned in the case that this instance is
+// not running in any debug mode.
+func (s *Server) debugMode() string {
+	switch {
+	case !s.devMode:
+		return ""
+	case os.Getenv("TS_DEBUG_LOGIN") == "on":
```

Comment:
Maybe this is just a `TS_DEBUG_WEB_MODE` envvar that has values "login" or "full" or something?  Related to above question... maybe this is going to end up being a relatively short term situation, in which case it hopefully won't matter too much what we do here.

But either way, the current names `TS_DEBUG_(LOGIN|FULL)` aren't quite enough to explain what these are or that they're even related to the web client.

Reply:
Oh I like the `TS_DEBUG_WEB_MODE` idea - I'll refactor to that! Maybe `TS_DEBUG_WEB_CLIENT_MODE`